python/tests/unittests/test_observe.py

```diff
@@ -282,9 +282,10 @@ def test_observe_multi_param(angle_0, angle_1, angles, want_state,
     with pytest.raises(RuntimeError) as error:
         # Too few.
         cudaq.observe(kernel, hamiltonian, np.pi, np.pi)
-    with pytest.raises(RuntimeError) as error:
-        # Too many list elements.
-        cudaq.observe(kernel, hamiltonian, np.pi, np.pi, [np.pi, np.pi, np.pi])
+    ## [SKIP_TEST]: Following check fails with "Failed: DID NOT RAISE <class 'RuntimeError'>"
```

Comment:
This may be a bad test - it is difficult to know from just looking at the MLIR exactly how many unique vector extractions there are, and then correlated that back to the number of elements the user provided.

Reply:
Removed in 99d9bee